webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/facts/nodes/NodeFactRepository.scala

```diff
@@ -190,12 +190,25 @@ trait NodeFactRepository {
    * Save (create or override) a core node fact
    * Use "updateInventory` if you want to save in pending, it's likely what you want.
    *
-   * Not that the diff is only done on the core properties
+   * Not that the diff is only done on the core properties.
+   * SecurityTag must not be updated with that method, but it can be initially set.
    */
   def save(
       nodeFact:  NodeFact
   )(implicit cc: ChangeContext, attrs: SelectFacts = SelectFacts.all): IOResult[NodeFactChangeEventCC]
 
+  /*
+   * A simpler version of save for CoreNodeFacts
+   */
+  def save(nodeFact: CoreNodeFact)(implicit cc: ChangeContext): IOResult[NodeFactChangeEventCC] = {
```

Comment:
this is just syntax sugar to avoid the gymnastic of the following line when what we have at hand is a `CoreNodeFact`.